sdk/data/wrapper/wrapper.go

```diff
@@ -280,3 +286,33 @@ func (w *Wrapper) updateDataNodeStatus() (err error) {
 
 	return
 }
+
+func (w *Wrapper) SetNearRead(nearRead bool) {
+	w.nearRead = nearRead
+	log.LogInfof("SetNearRead: set nearRead to %v", w.nearRead)
+}
+
+func (w *Wrapper) GetNearRead() bool {
+	return w.nearRead
+}
+
+// Sort hosts by distance form local
+func (w *Wrapper) sortHostsByDistance(hosts []string) []string {
+	var tmpHost string
+	for i := 0; i < len(hosts); i++ {
+		for j := i + 1; j < len(hosts); j++ {
+			if distanceFromLocal(hosts[i]) > distanceFromLocal(hosts[j]) {
+				tmpHost = hosts[i]
```

Comment:
use host[i],host[j]=host[j],host[i] replace it

Reply:
resolved